sources/packages/web/src/types/contracts/ApiProcessError.ts

```diff
@@ -1,7 +1,6 @@
 export const MORE_THAN_ONE_APPLICATION_DRAFT_ERROR =
   "ONLY_ONE_APPLICATION_DRAFT_PER_STUDENT_ALLOWED";
 
-export interface ApiProcessError {
-  message: string;
-  errorType: string;
+export class ApiProcessError {
```

Comment:
Why did we change this to class?

Reply:
Is that to use the instanceof inside catch?

Reply:
Yes, it is.